docs/Telemetry.md

```diff
@@ -106,6 +106,15 @@ _Subscribing Event_:
     }, []);
 ```
 
+### Broadcast Events
+
+Refer to the below table to understand different broadcast events raised during the lifetime of the chat widget.
```

Comment:
Are there more of these non-telemetry event that we raise? Please log them all here if any.